### PR TITLE
SAK-49483 Kernel allow removal of empty dropbox folders for non existing users

### PR DESCRIPTION
**Pull request by danielmerino**

### CODE REVIEW

File: kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxAuthzHandler.java [hunk 1]
```diff
@@ -314,4 +314,17 @@ private static String getDropboxOwner(String entityId)
...
+	
+	private static boolean isSiteMember(String siteId, String userId)
+	{
+		try
+		{
+			return siteService.getSite(siteId).getMember(userId) != null;
+		}
+		catch (IdUnusedException e)
+		{
+			log.warn("IdUnusedException in isSiteMember for " + userId);
```
Comment (ern) [inline kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxAuthzHandler.java:326]: better log message and use newer log pattern and don't forget to add the exception message for better logging
```suggestion
			log.warn("User [{}] is not a member of site [{}], {}", userId, siteId, e.toString());
```

Comment (danielmerino) [reply kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxAuthzHandler.java]: Sorry for the delay, I have been on holidays. Thanks for the review, Earle, the change is done.

Comment (danielmerino) [reply kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxAuthzHandler.java]: Hi @ern , do you have any other suggestion on this code? Can it be merged? Thanks!

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Well done Dani @danielmerino

**Pull request merged**